eahub/profiles/forms.py

```diff
@@ -61,18 +67,20 @@ class Meta:
             'open_to_job_offers',
         )
         labels = {
-            'open_to_job_offers': ('Open to job offers:')
+            'open_to_job_offers': ('Open to job offers')
         }
 
-
 class EditProfileCommunityForm(forms.ModelForm):
+    groups = CustomisedMultipleModelChoiceField(queryset=LocalGroup.objects.all(), required=False, label="EA Groups")
```

Comment:
When you look at the HTML of a form generated this way, does it include the database IDs of the groups? If so, we should prevent this by setting `to_field_name` to `slug`.

Reply:
No, the html only shows the group's name
![screen shot 2019-03-04 at 20 11 39](https://user-images.githubusercontent.com/15908208/53760096-ca92c900-3eb9-11e9-8dad-17b34e8a7676.png)